src/ucp/rndv/rndv.c

```diff
@@ -867,6 +867,7 @@ ucp_rndv_send_frag_get_mem_type(ucp_request_t *sreq, ucs_ptr_map_key_t rreq_id,
                                                        : UCP_NULL_RESOURCE;
     }
 
+    freq->status = UCS_INPROGRESS;
```

Comment:
is it possible freq is already released, in line 872?

Reply:
nice catch! but actually this value is never used, need to return void